addons/dragging/XEH_postInit.sqf

```diff
@@ -15,6 +15,9 @@ if (isNil "ACE_maxWeightDrag") then {
 if (isNil "ACE_maxWeightCarry") then {
     ACE_maxWeightCarry = 600;
 };
+if (isNil QGVAR(maxWeightCarryRun)) then {
+    GVAR(maxWeightCarryRun) = 50;
```

Comment:
This seems a bit low. Max weight of 10lbs (`FUNC(getWeight) multiplies `mass` by 0.5) while running? 100 would be more appropriate, I think.